Fitness/Views/AllProRoutinesView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  AllProRoutinesView.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/21/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    func addRoundedCornersAndShadows(radius: CGFloat) {
+        layer.cornerRadius = radius
+        layer.masksToBounds = false
+    }
+}
+
+extension CALayer {
```

Comment:
Can these extensions be put in a CALayer+Extensions file and a UIView+Extensions file? Or if they're not used anywhere but with one UIView type we could add it as a function of that view instead.

Reply:
I'll make these functions of the view for now